src/Agent.Worker/Build/BuildJobExtension.cs

```diff
@@ -29,25 +29,33 @@ public BuildJobExtension()
                 @finally: false);
         }
 
-        private async Task<TaskResult> PrepareAsync()
+        private Task PrepareAsync()
```

Comment:
I removed TaskResult from being returned by the job-plugin methods. It's already on the execution context (which the job plugin has access to).